flash_tutorials/text_classification/text_classification.py

```diff
@@ -46,7 +44,7 @@
     train_file="data/imdb/train.csv",
     val_file="data/imdb/valid.csv",
     test_file="data/imdb/test.csv",
-    batch_size=4,
+    batch_size=512,  # just increased for the example to run fast
```

Comment:
@Borda - I had to increase the `batch_size`, as the dataset is huge - and with 512 batch_size, there will be around 40-50 iterations. (with 4, there were more than 1500).

Reply:
:+1: